pom.xml

```diff
@@ -44,18 +56,6 @@
     <version.org.hawkular.bus>0.3.3.Final</version.org.hawkular.bus>
   </properties>
 
-  <modules>
-    <module>api</module>
-    <module>hawkular-inventory-json-helper</module>
-    <module>impl-tinkerpop-parent</module>
-    <module>hawkular-inventory-cdi</module>
-    <module>rest-servlet</module>
-    <module>bus</module>
-    <module>bus-api</module>
-    <module>hawkular-integrated-inventory-rest</module>
```

Comment:
after we release the change will be needed also on `hawkular/hawkular` side (to depend on `hawkular-inventory-dist` instead of `hawkular-integrated-inventory-rest`)